### DISCUSSION TITLE
Question 1

### DISCUSSION THREAD

**Discussion by Zakiyyyy** [Zakiyyyy/my-repo#3] [Q&A] [answered]
How to add css eksternal in my project?

**Comment by Kholik-Muzaki** [comment, marked as answer]
To add external CSS to a Bootstrap boilerplate, start by creating a CSS file named `styles.css` (or any name you prefer) and place it in a directory within your project, such as `css` or `assets/css`. Add your custom CSS rules to this file. Then, open your HTML file and link the external CSS file within the `<head>` section, ensuring that this link is placed after the Bootstrap CSS link. This placement ensures that your custom styles will override Bootstrap’s styles if necessary. Finally, open your HTML file in a browser to verify that your custom styles are correctly applied alongside Bootstrap’s styles.

**Comment by Kholik-Muzaki** [comment]
To add external CSS to a Bootstrap boilerplate, start by creating a CSS file named `styles.css` (or any name you prefer) and place it in a directory within your project, such as `css` or `assets/css`. Add your custom CSS rules to this file. Then, open your HTML file and link the external CSS file within the `<head>` section, ensuring that this link is placed after the Bootstrap CSS link. This placement ensures that your custom styles will override Bootstrap’s styles if necessary. Finally, open your HTML file in a browser to verify that your custom styles are correctly applied alongside Bootstrap’s styles.
